bounties_api/bounties/settings.py

```diff
@@ -241,15 +241,15 @@
 GITHUB_TOKEN = os.environ.get('github_token', '')
 SNS_ADDRESS = os.environ.get('sns_address', '')
 
-networks = {
+NETWORKS = {
```

Comment:
Is this used anywhere else in the application? I don't think it actually is... but just double-checking if you did a search?

Reply:
I did a search and didn't find it anywhere else